demo/src/App.js

```diff
@@ -78,7 +94,7 @@ class App extends Component {
       nodeSize: { x: 200, y: 200 },
       enableLegacyTransitions: false,
       transitionDuration: 500,
-      renderCustomNodeElement: customNodeFnMapping['svg'].fn,
+      renderCustomNodeElement: customNodeFnMapping['input'].fn,
```

Comment:
Thanks for adding an example for the `interactiveNode` use case 🙏  

The default for the demo should still be the simplest `svg` example, so `input` can be one of the other options:
```suggestion
      renderCustomNodeElement: customNodeFnMapping['svg'].fn,
```